ibm_catalog.json

```diff
@@ -57,6 +57,180 @@
               }
             ]
           },
+          "configuration" : [
+            {
+              "key": "ibmcloud_api_key"
+            },
+            {
+              "key": "use_existing_resource_group"
+            },
+            {
+              "key": "resource_group_name"
+            },
+            {
+              "key": "region",
+              "options": [
```

Comment:
since SM is supported in all regions, we don't have to list all the regions like this. We can use this pre-defined picker:
```
{
  "custom_config": {
    "config_constraints": {
      "generationType": "2"
    },
    "grouping": "deployment",
    "original_grouping": "deployment",
    "type": "region"
  },
  "key": "region",
  "required": true,
  "type": "string"
}
```